src/mumble/GlobalShortcut_win.cpp

```diff
@@ -899,3 +902,24 @@ QString GlobalShortcutWin::buttonName(const QVariant &v) {
 bool GlobalShortcutWin::canSuppress() {
 	return bHook;
 }
+
+bool areScreenReadersActive() {
```

Comment:
I would declare this function into the `GlobalShortcutWin` class.

Reply:
Don

Reply:
All done.